### ISSUE TITLE
Add lightwalletd configuration enabled information to getinfo output

### ISSUE THREAD

**Issue by LarryRuane** [zcash/zcash#4585]
Currently, a lightwalletd has no way to know if the zcashd it's connected to is properly configured to support lightwalletd clients. (It must be running with `-lightwalletd` and `-txindex` enabled.) If the zcashd isn't configured properly, then (for example) when a wallet queries for a transaction, it will receive a "not found" error, but it has no way to know whether it's due to zcashd misconfiguration or if the transaction truly doesn't exist.

Instead of adding a new RPC, we can add a boolean "lightwalletd" to the existing `getinfo` RPC. This would be a very minimal amount of work (just a couple of lines).

Once this is done, a separate ticket can request that the lightwalletd use this information to fail on startup with a clear error message.

Thanks to @benzcash for suggesting this idea.

We may want to make a similar change for `-insightexplorer`, although maybe not, since there's no equivalent to lightwalletd when a zcashd is acting as an explorer endpoint.
